src/DesignQuery/DesignQuery.cpp

```diff
@@ -171,5 +883,142 @@ bool DesignQuery::RegisterCommands(TclInterpreter* interp, bool batchMode) {
     return (status) ? TCL_OK : TCL_ERROR;
   };
   interp->registerCmd("get_ports", get_ports, this, 0);
+
+  //  // first SDT API function implementation
```

Comment:
Please remove/add as needed.

Reply:
Added functionality of get_file_ids tcl command. Needed to give correct path to hier_info.json file for this tcl command to run.